### DISCUSSION TITLE
can`t save preview because spaces in file name

### DISCUSSION THREAD

**Discussion by fin-gal-25** [AlpacaInTheNight/PromptsBrowser#10] [Ideas]
I have spaces and other symbols in path to file, save preview didn`t work, so I have modify "src" in savepreview.py

```
from urllib.parse import unquote
src = unquote(postJSON.src)

```

**Comment by AlpacaInTheNight** [comment]
Indeed such a problem exists with certain setups and it is necessary to use .unquote method. I will add this in the next update.

On which OS and browser did this problem occur?

Oh, never mind. I think I found when this occurs. My setup names generated images by numbers, so I don't have special chars encoding issue. But on setups that uses prompts in names this issue will occur. This is a serious flaw I have missed.
